index.bs

```diff
@@ -324,6 +324,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Resident Key</dfn>
+: <dfn>RK</dfn>
+:: A [=resident key=] is a [=credential private key=] that can be found and used by the authenticator based on only the 
+    associated RP ID. 
+
```

Comment:
It might help with naming to focus on what this field means for calls to `.get()`. As Vijay [said](https://github.com/w3c/webauthn/pull/378#issuecomment-288467611), if this field is omitted or `false`, the RP must pass a credential ID in [`allowList`](https://w3c.github.io/webauthn/#dom-scopedcredentialrequestoptions-allowlist). If it's `true`, then the RP doesn't need to pass the credential ID. So `mayOmitCredentialId` or `storeCredentialId` could be good names, or if you're willing to invert the default, `willPassCredentialId` or `getRequiresCredentialId`. This is used as `create({publicKey: {storeCredentialId: true}})`, right?

Allowing sites to omit the credential ID also allows the site to rely on browser or authenticator UI to select which account to log into (for a first-factor authenticator), while requiring the site to pass it forces the site to either use the authenticator as a second factor or to have the user explicitly type their username. Given that, maybe `firstFactor` or `authenticatorCanSelectAccount`, although I don't love either of those.